test/e2e/content-header.test.js

```diff
@@ -0,0 +1,26 @@
+
+const { assert, driver } = require('vl-ui-core').Test;
+const VlContentHeaderPage = require('./pages/vl-content-header.page');
+
+describe('vl-content-header', async () => {
```

Comment:
Extra test voorzien om img webcomponent (page object) te bekomen. In de test kan je dan enkele attributen van de image controleren ter validatie dat alles werkt.

Reply:
Toegevoegd